test/integration/exasol/test_regression.py

```diff
@@ -42,15 +41,16 @@ def setup_class(cls):
                 Column("name", String(1000), nullable=False),
                 schema=cls.tenant_schema_name,
             )
-            metadata.create_all(engine)
+            metadata.create_all(conn)
 
     @classmethod
     def teardown_class(cls):
-        with config.db.connect() as conn:
+        with config.db.begin() as conn:
             conn.execute(DropSchema(cls.tenant_schema_name, cascade=True))
 
     def test_use_schema_translate_map_in_get_last_row_id(self):
         """See also: https://github.com/exasol/sqlalchemy-exasol/issues/104"""
+        pass
```

Comment:
```suggestion
```

Reply:
Ah thanks - I was wondering about this line, too.